java/src/jmri/jmrix/sprog/SprogReply.java

```diff
@@ -268,6 +282,8 @@ public boolean endBootReply() {
         }
     }
 
+    // deprecated since 4.11.4 as bootloading old sprogs is no longer supported in JMRI
```

Comment:
Please use the `@deprecated` tag in Javadocs to document when the deprecation occurred. This allows the Javadocs to display a deprecation notice.
Something like the following would work:
```
/**
 * @deprecates since 4.11.4; bootloading old SPROGs is no longer supported in JMRI
 */
```
See the note on "Deprecating Code" in http://jmri.org/help/en/html/doc/Technical/RP.shtml